docs/trace/getting-started/README.md

```diff
@@ -50,15 +50,39 @@ What does the above program do?
 
 The program creates an `ActivitySource` which represents an [OpenTelemetry
 Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracer).
+
+```csharp
+private static readonly ActivitySource MyActivitySource = new ActivitySource(
+        "MyCompany.MyProduct.MyLibrary");
+```
+
 The `ActivitySource` instance is used to start an `Activity` which represents an
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span).
+
+```csharp
+using (var activity = MyActivitySource.StartActivity("SayHello"))
+      {
+          activity?.SetTag("foo", 1);
+          activity?.SetTag("bar", "Hello, World!");
+          activity?.SetTag("baz", new int[] { 1, 2, 3 });
+      }
+```
+
 An OpenTelemetry
 [TracerProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracerprovider)
 is configured to subscribe to the activities from the source
 `MyCompany.MyProduct.MyLibrary`, and export it to `ConsoleExporter`.
 `ConsoleExporter` simply displays it on the console.
 
+```csharp
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
A question about the doc structure, is it better to doc `TracerProvider` before `ActivitySource`, as it represents SDK is is more basic than `Tracer`?